README.md

```diff
@@ -53,6 +53,36 @@ and Erlang/OTP.
 
 **Note** *: prior to 23, Windows builds are only available for minor versions, e.g. 21.0, 21.3, 22.0, etc.
 
+### Self-hosted runners
+
+We try to guess the self-hosted runner _container_ equivalent (to download assets used in the action)
+by making direct calls to the operating system, but this might fail, in which case we
+warn with an error message, and you should set environment variable `ImageOS` to one of the following:
```

Comment:
Still need to run a short test using this action to make sure the error message is not an "unhandled promise".

Reply:
Test is done. Error is caught as expected.